packages/core/lib/commands/call/run.js

```diff
@@ -0,0 +1,317 @@
+module.exports = async function (options) {
+  const debug = require("debug")("core:commands:call");
+  const fs = require("fs");
+  const util = require("util");
+  const { Environment } = require("@truffle/environment");
+  const OS = require("os");
+  const Codec = require("@truffle/codec");
+  const Encoder = require("@truffle/encoder");
+  const Decoder = require("@truffle/decoder");
+  const TruffleError = require("@truffle/error");
+  const { fetchAndCompile } = require("@truffle/fetch-and-compile");
+  const loadConfig = require("../../loadConfig");
+  const DebugUtils = require("@truffle/debug-utils");
+  const web3Utils = require("web3-utils");
+
+  if (options.url && options.network) {
+    const message =
+      "" +
+      "Mutually exclusive options, --url and --network detected!" +
+      OS.EOL +
+      "Please use either --url or --network and try again." +
+      OS.EOL +
+      "See: https://trufflesuite.com/docs/truffle/reference/truffle-commands/#call" +
+      OS.EOL;
+    throw new TruffleError(message);
+  }
+
+  let config = loadConfig(options);
+  await Environment.detect(config);
+
+  const [contractNameOrAddress, functionNameOrSignature, ...args] = config._;
+  let functionEntry, transaction;
+
+  const fromAddress =
+    options.from ?? config.networks[config.network]?.from ?? config.from;
+  if (!web3Utils.isAddress(fromAddress)) {
```

Comment:
Can we skip the need to specify a `from` field?

At the very least this error should indicate which address is the problem, since this error now suggests that it might be the contract address that's wrong

Reply:
But I'm thinking... maybe we can just use `0x0` as the `from` address